.gitlab-ci.yml

```diff
@@ -0,0 +1,20 @@
+image: dockerhub.ebi.ac.uk/ega/ega-data-api-build-container:latest
+
+services:
+  - docker:dind
+
+stages:
+  - build
+  - package
+
+maven-build:
+  stage: build
+  script:
+    - mvn test
+
+docker-build:
+  stage: package
+  script:
+    - mvn clean install "-DdockerRegistry=$CI_REGISTRY_IMAGE" "-Dimage.version=$CI_COMMIT_TAG" -P production -DskipTests
```

Comment:
Do we need 'mvn clean install' here or 'mvn clean package' will also do?

Reply:
changed to package